### PR TITLE
Storybook: added new story for chat demostrating typingUsers

### PR DESCRIPTION
**Pull request by Strider2342**

### CODE REVIEW

**Comment by Strider2342** [comment]
[Chat: Add a new story to the Storybook [only typingUsers?]](https://trello.com/c/pWwdSwBW/9556-chat-add-a-new-story-to-the-storybook-only-typingusers)

File: apps/react-storybook/stories/chat/Chat.stories.tsx [hunk 1]
```diff
@@ -390,3 +390,47 @@ export const Customization: Story = {
...
+                'No one is typing',
+                'One user typing',
+                'Two users typing',
+                'Three users typing',
+            ],
+            mapping: {
+                ['No one is typing']: [],
+                ['One user typing']: [ firstAuthor ],
+                ['Two users typing']: [ firstAuthor, secondAuthor ],
+                ['Three users typing']: [ firstAuthor, secondAuthor, thirdAuthor ],
```
Comment (EugeniyKiyashko) [inline apps/react-storybook/stories/chat/Chat.stories.tsx:413]: The scenario whenMultiple Users are typing:  "{0} and others are typing..." is missed

File: apps/react-storybook/stories/chat/Chat.stories.tsx [hunk 2]
```diff
@@ -390,3 +390,47 @@ export const Customization: Story = {
...
+export const TypingUsers: Story = {
+    args: {
+        width: 400,
+        height: 500,
+        typingUsers: [],
+    },
+    argTypes: {
+        typingUsers: {
+            control: 'select',
+            defaultValue: 'No one is typing',
```
Comment (EugeniyKiyashko) [inline apps/react-storybook/stories/chat/Chat.stories.tsx]: After loading the story, I would prefer the value 'One user typing' to be selected by default in the select box, and for the demo to reflect this choice.

File: apps/react-storybook/stories/chat/Chat.stories.tsx (hunk 1 above)
Comment (Strider2342) [reply apps/react-storybook/stories/chat/Chat.stories.tsx:413]: e71c9b02ce630f5a790eacc297e696b421101a5e

File: apps/react-storybook/stories/chat/Chat.stories.tsx (hunk 2 above)
Comment (Strider2342) [reply apps/react-storybook/stories/chat/Chat.stories.tsx]: e71c9b02ce630f5a790eacc297e696b421101a5e

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by Zedwag — APPROVED** [review]

**Pull request merged**